doc/sphinx/Clusters_from_Scratch/installation.rst

```diff
@@ -7,7 +7,8 @@ Install |CFS_DISTRO| |CFS_DISTRO_VER|
 Boot the Install Image
 ______________________
 
-Download the 8.9GB |CFS_DISTRO| |CFS_DISTRO_VER| `DVD ISO <http://mirror.metrocast.net/centos/8-stream/isos/x86_64/CentOS-Stream-8-x86_64-20201211-dvd1.iso>`_.
+Download the latest |CFS_DISTRO| |CFS_DISTRO_VER| DVD ISO for the architecture of your choice
+by navigating to the correct subdirectory within `this repository <http://mirror.metrocast.net/>`_.
```

Comment:
Playing around with the links a bit, I'm thinking the most useful for users will be http://isoredirect.centos.org/centos/8-stream/isos/x86_64/ , to let people pick the mirror closest to them (and in case the one we would pick is down). Unfortunately the redirect URL only works with an architecture specified, but most people want x86_64 and those that don't should already be familiar with finding the right one.